docs/source/gettingstarted.rst

```diff
@@ -12,19 +12,19 @@ Installation
 
 Datalad is a Python package and can be installed via pip_, which is the
 preferred method unless system packages are available for the target platform
-(see below)::
+(see below). To automatically install datalad and all its software dependencies
+type::
 
   pip install datalad
 
 .. _pip: https://pip.pypa.io
 
-This will automatically install all software dependencies necessary to provide
-core functionality. Several additional installation schemes are supported
-(e.g., ``publish``, ``metadata``, ``tests``, ``crawl``)::
+Several additional installation schemes are supported (``[SCHEME]`` can be e.g.
+``publish``, ``metadata``, ``tests`` or ``crawl``)::
 
-  pip install datalad[SCHEME]
-
-where ``SCHEME`` can be any supported scheme, such as the ones listed above.
```

Comment:
I would even say that this "expert info", and we should have a clearly separate "I am new here, and I am not a python expert" intro section. Not intermingling everything. But this can be done later.